src/js/renderers/editor-dom.js

```diff
@@ -43,6 +44,9 @@ function renderHTMLText(marker) {
   text = text.replace(/ ( )/g, () => {
     return ' '+NO_BREAK_SPACE;
   });
+  text = text.replace(/\t/g, () => {
```

Comment:
If the editor used/exported its `TAB_CHARACTER` constant ('\t') then this regex could also use it. Seems a bit less brittle than to hardcode "\t" in the two different files.